### ISSUE TITLE
Add sUSD rate to avoid 1:1 ratio with USD

### ISSUE THREAD

**Issue by clementbalestrat** [Synthetixio/synthetix-mintr#52]

**Comment by bachstatter** [comment]
It seems like it is already using the sUSD rate:
https://github.com/Synthetixio/synthetix-mintr/blob/staging/src/screens/Dashboard/Dashboard.js#L110
Multiplying the balance with the price.

But `snxJSConnector.snxJS.ExchangeRates.ratesForCurrencies` (inside `fetchData.js`) gives back a sUSD rate of 1 even though https://coinmarketcap.com/currencies/susd/ has it at ~0.99

Not sure whats going on there.

**Comment by bachstatter** [comment]
Also should the Synths and the Debt really be multiplied with the sUSD price?
I think NO but wanted to confirm in case I'm missing something.
https://github.com/Synthetixio/synthetix-mintr/blob/staging/src/screens/Dashboard/Dashboard.js#L114

**Comment by justinjmoses** [comment]
No is right - I believe it’s just an oversight right @clementbalestrat? It should show in sUSD terms

**Comment by bachstatter** [comment]
> No is right - I believe it’s just an oversight right @clementbalestrat? It should show in sUSD terms


The table column header should probably be sUSD instead of USD then?

**Comment by hav-noms** [comment]
Yeah this is a tricky one. @kaiynne wants mintr to display the sUSD market price but the whole system works of $1 peg value to issue and burn. I think it would be misleading to users to show the market price unless the system used it which it does not. @kaiynne what do you want to do here?

**Comment by clementbalestrat** [comment]
I feel like a dashboard is supposed to show you the real value of your assets, so maybe USD should be used here. If not maybe we should think about another way to display this table as sUSD, Synths and Debt row values are just duplicates.

**Comment by clementbalestrat** [comment]
@bachstatter @hav-noms Kain is happy with USD.

@hav-noms any idea what's the best source to get sUSD/USD rate?

**Comment by justinjmoses** [comment]
OK, but considering the best liquidity is via the sETH pool on uniswap, should not that rate be used? Price everything in sETH terms, use the sETH to ETH rate from the pool, and then ETH to USD. It's more complicated, but it's more reliable than the sUSD price IMO.

**Comment by clementbalestrat** [comment]
Good point.

But then the whole Mintr Dashboard will be tied to Eth price fluctuation. Does it make sense to have your balances decreasing by 5% if ETHUSD drops by 5%?

**Comment by justinjmoses** [comment]
As explained in person it doesn’t matter because we multiply by the rate then divide by it. 

E.g Assuming sETH:ETH is 0.997 
when ETHUSD is 100:1, 100 sUSD is 1 sETH = 0.997 ETH = 99.7 USD
When ETHUSD doubles, 100 sUSD is 0.5 sETH = 0.4985 ETH = 99.7 USD

**Comment by clementbalestrat** [comment]
@bachstatter, so here are the steps:

- balances in sUSD
- get sUSD/sETH exchange rate from our ExchangeRates smart contract (I can show you how to do to it)
- get sETH/ETH exchange rate from Uniswap API (https://uniswap-api.loanscan.io/v1/ticker?exchangeAddress=0xe9cf7887b93150d4f2da7dfc6d502b216438f244). `invPrice is probably the value to pick.
- get ETH/USD exchange rate from our ExchangeRates smart contract (the value is already in the code)
